EIPS/eip-6800.md

```diff
@@ -175,7 +175,7 @@ Bytes `1..4` are reserved for future use.
 
```

Comment:
General note: in this EIP I'm not mentioning any changed strategy on how code-chunking is performed.

I think we have three options:
- Use 31-byte chunker both for legacy and EOFv1 contracts. This is the simplest but inefficient for EOF, so see very low chances of keeping status quo.
- Use 32-byte chunker for both legacy and EOFv1. For legacy we need a new "jump table" to mark invalid jumps, which adds some complexity.
- Use a 31-byte chunker for legacy, and 32-byte chunker for EOFv1. This might be the simplest one in the sense that no new "jump table" is required for legacy and also get the efficiency of 32-byte chunker for EOFv1. But, the EL client needs to have an `if` to know which kind of chunker to use depending on which contract is "active".

I preferred to leave this discussion out of this PR, and we can jump again into this topic in a future VIC and adjust the EIPs accordingly.